packages/common/src/discussions/edit.ts

```diff
@@ -126,13 +127,34 @@ export async function updateDiscussion(
   let updatedDiscussion = mapper.storeToEntity(updatedModel, discussion);
   updatedDiscussion = computeProps(model, updatedDiscussion, requestOptions);
 
+  // persist location geometries to discussion settings as Geometry[]
+  let allowedLocations;
+  try {
+    allowedLocations =
+      updatedDiscussion.location?.geometries?.map((geometry) =>
+        arcgisToGeoJSON(geometry)
+      ) || null;
+  } catch (e) {
+    /* istanbul ignore else */
+    if (e instanceof Error) {
+      allowedLocations = null;
+      /* tslint:disable no-console */
+      console.warn("Esri JSON conversion failed:", e.message);
+    } else {
+      // do nothing
+    }
+  }
+
   // create or update entity settings
   const defaultSettings = getDefaultEntitySettings("discussion");
   const settings = {
     ...defaultSettings.settings,
     discussions: {
       ...defaultSettings.settings.discussions,
-      ...updatedDiscussion.discussionSettings,
+      ...{
```

Comment:
Do we need to wrap this in an object? Curious if this could be simplified to just:

```tsx
  const settings = {
    ...defaultSettings.settings,
    discussions: {
      ...defaultSettings.settings.discussions,
      ...updatedDiscussion.discussionSettings,
      allowedLocations,
    },
  };
```